ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -312,6 +314,7 @@ measurement:
   rotationalvelocity: "Vector measure of rotation rate, that refers to how fast an object rotates or revolves relative to another point."
   specificenergy: "Energy per unit mass."
   specificenthalpy: "Measure of energy of air per unit mass."
+  soundintensitylevel: "Logarithmic measure of the sound intensity in comparison to a reference value (1e10-12 W/m^2)."
```

Comment:
Please name this `soundpressurelevel` to match QUDT (https://qudt.org/vocab/quantitykind/SoundPressureLevel)